packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.routing.js

```diff
@@ -43,19 +48,28 @@ export default /* @ngInject */ ($stateProvider) => {
 
       volumeBackupModel: /* @ngInject */ (
         preselectedVolume,
+        preselectedVolumeId,
         preselectedVolumeOption,
         prefilledBackupName,
         PciProject,
         customerRegions,
+        volumes,
       ) => {
```

Comment:
```js
if (preselectedVolumeId) {
  preselectedVolume = volumes?.find((volume) => volume.id === preselectedVolumeId);
}
```

can't you just add this at the top of the function ? i think it should be enough

Reply:
I can simplify but i can't reassign preselectedVolume because it's a parameter ( eslint error)